donor-portal/src/components/navbar/styles.css

```diff
@@ -136,13 +136,47 @@
 }
 
 .nav-link {
-  margin: 18px;
   color: white;
-  font-size: 14px;
+  padding: 0 15px;
+  font-size: 16px;
+  letter-spacing: 3px;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  position: relative;
+  transition: all 0.5s;
+  z-index: 2;
+}
+
+.nav-link::before {
+  content: attr(a);
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  bottom: -50%;
+  left: 0;
+  z-index: -1;
+  transition: all 0.5s;
+  transform: rotate3d(1, 0, 0, 90deg);
+  background-color: #fff;
 }
+
+.nav-link:hover::before,
+.nav-link.active::before {
+  bottom: 0;
+  transform: rotate3d(1, 0, 0, 0deg);
+}
+
+.nav-link:not(:last-of-type) {
+  border-right: 1px solid white;
+}
+
+.nav-link.active,
 .nav-link:hover {
-  color: var(--elixir-color-gray-400);
+  color: var(--elixir-color-gray-900);
+  /* padding: 2rem; */
```

Comment:
to remove?

Reply:
gray 400 isn't dark enough to match the contrast of #fff (on this specific design)

Reply:
Oh, that's completely fine. I was asking about the commented out line, is it to be removed?